products/compute/api.yaml

```diff
@@ -779,8 +779,6 @@ objects:
                 either maxRate or maxRatePerEndpoint must be set.
             - !ruby/object:Api::Type::Double
               name: 'maxUtilization'
-              send_empty_value: true
-              default_value: 0.8
```

Comment:
The docs here still says the default is 0.8. Also, is it possible that removing this default causes diffs (since a user's state file might say 0.8 is the value but their config says it's nothing)?

Reply:
Due to an interesting behavior (bug?) in backend service, `max_utilization` never is used in the hash function for backend: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/constants/backend_service.go.erb

So modifying this field never causes a diff. I'm worried that adding this field to the hash function would be a breaking change

Reply:
Also, because this is moving to Optional+Computed it shouldn't cause a diff if their config says nothing (assuming the hash function was working correctly)

Reply:
https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3764-old..auto-pr-3764#diff-5634ebdaad9fa3c2126514df6c5cb870R918 doesn't show it as O+C

Reply:
Ah, right it doesn't come back from the API in most cases so just optional works. 

Keeping `send_empty_value` makes terraform send a value in every circumstance, meaning that we can't *not* specify this field for NEG backends. I could strip it from requests for NEG backends with a virtual field, but then it still appears in state and create plans.

For some reason I'm seeing zero values be sent even with `send_empty_value: false`. Not entirely sure why that is happening